enarxbot-pr-triage

```diff
@@ -43,7 +43,12 @@ result = bot.graphql(
     }
     """,
     pr=pr,
-    page=["node", "projectCards"]
+    cursors= {
+        'cursor': {
+            'path': ["node", "projectCards"],
+            'next': None
```

Comment:
You should be able to make this field optional with `cursors[cursor].get('next')`.